app/assets/javascripts/naming_vote_ajax.js

```diff
@@ -1,86 +1,36 @@
+// Voting can be slow, so this script disables other form elements
+// while waiting for an AJAX response.
+
 function VoteByAjaxModule(translations) {
   $(document).ready(function () {
 
-    var _haveVotesChanged = false;
-    var _haveVotesBeenSaved = false;
-
     var save_vote_buttons = function () {
       return $("[data-role='save_vote']");
     };
 
+    // Don't need these if AJAX available.
+    save_vote_buttons().hide();
+
     var change_vote_selects = function () {
       return $("[data-role='change_vote']");
     };
 
     var attach_bindings = function () {
       change_vote_selects().on("change", function (event) {
-        var _this = $(this);
-        var value = _this.val();
-        var naming_id = _this.data("id");
-        _haveVotesChanged = true;
-
-        // If setting vote to 3.0, go through all the rest and downgrade any
-        // old 3.0's to 2.0.  Only one 3.0 vote is allowed. Also disable all
-        // the menus while the AJAX request is pending.
-        if (value == "3.0") {
-          change_vote_selects().each(function () {
-            var _this2 = $(this);
-            if (_this2.data("id") != naming_id && _this2.val() == "3.0") {
-              _this2.val("2.0");
-            }
-            _this2.attr("disabled", "disabled");
-          });
-        }
-
```

Comment:
Any particular reason to do away with this behavior?

Reply:
Yes - sorry I modified the PR comment after requesting your review, it probably didn't go out in the email.

> - It doesn't have to downgrade other selects if the user votes "I'd call it that", because the whole naming table will get refreshed if the ajax request is successful
> - I don't believe we need to handle saving a backup of the select "old value" data on the event of unsuccessful AJAX. The selects just won't be changed by a naming table refresh.

Reply:
Ah, right.  Okay, then the only difference is that we are no longer disabling the controls while waiting... is that also covered by the new "fire" thingy?

Reply:
Yes, but it's handled a different way. Now, the "Saving... whirly" modal disables all events on screen.

I was surprised to find that on the current live version, you can dismiss the "saving" modal while you're waiting, but in my book, there's no need to interact with the page at all.